package org.telegram.ui.Cells;

import android.widget.FrameLayout;

/* compiled from: r8-map-id-e506a87262d42a59d49ceeb11de21243ca58d8dd989db9ff2eb23aa08d8dd348 */
/* loaded from: classes3.dex */
public final class d7 extends org.telegram.ui.Components.f9 {
    public final /* synthetic */ int D;
    public final /* synthetic */ FrameLayout E;

    /* JADX WARN: 'super' call moved to the top of the method (can break code semantics) */
    public /* synthetic */ d7(FrameLayout frameLayout, org.telegram.ui.ActionBar.e6 e6Var, int i10) {
        super(e6Var);
        this.D = i10;
        this.E = frameLayout;
    }

    @Override // android.graphics.drawable.Drawable
    public final void invalidateSelf() {
        switch (this.D) {
            case 0:
                super.invalidateSelf();
                ((f7) this.E).a.invalidate();
                break;
            default:
                super.invalidateSelf();
                ((g7) this.E).a.invalidate();
                break;
        }
    }
}
